lib/lookup.json

```diff
@@ -328,7 +328,8 @@
   },
   "path-to-regexp": {
     "prefix": "v",
-    "maintainers": "blakeembrey"
+    "maintainers": "blakeembrey",
+    "skip": ["aix"]
```

Comment:
FWIW I in the (slow) process of PR'ing a fix to Jest for this (and other modules that fail on AIX that use Jest): https://github.com/facebook/jest/issues/10263

Reply:
I included reference to why it was skipping in the commit comment so we should be able to grep / identify / unskip when this is fixed